### ISSUE TITLE
Expose marker alpha option

### ISSUE THREAD

**Issue by cammace** [mapbox/mapbox-gl-native#5279]
Looks like we give the [option to set the marker alpha](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerView.java#L293-L316) when creating a view adapter but don't expose the API in `MarkerViewOptions()`. 

Should look something like this:
```java
MarkerViewOptions marker = new MarkerViewOptions()
                        .position(new LatLng(-33.85699436, 151.21510684))
                        .alpha(0.5f);
```

@tobrun @bleege @zugaldia @ivovandongen

**Comment by tobrun** [comment]
Great catch @cammace!

**Comment by tobrun** [comment]
This landed in 7942ce9 in the `android-v4.1.0` release branch. Closing
